packages/create-plugin/templates/app/provisioning/plugins/apps.yaml

```
@@ -1,5 +1,11 @@
 apiVersion: 1
 
 apps:
-  - type: '{{ normalize_id pluginName orgName 'app' }}'
+  - type: '{{ normalize_id pluginName orgName 'app' }}')
+    org_id: 1
+    org_name: Main Org.
```

Let's use template variables here as well.
```suggestion
    org_name: '{{ orgName }}'
```